src/core/batch/parquetSchema.ts

```diff
@@ -80,6 +102,39 @@ export const ReactionSchema = {
   signature: { type: "BYTE_ARRAY" },
 };
 
+/**
+ * ReactionBloomFilter: bloom filter options for batching reaction messages
+ */
 export const ReactionBloomFilterOptions = {
   bloomFilters: [{ column: "emoji" }, { column: "fromId" }, { column: "inReplyTo" }],
 };
+
+/**
+ * getSchemaFor() takes DSNPType and returns its corresponding parquet schema
+ *
+ * @param   dsnpType a dsnpType
+ * @returns The corresponding parquet schema
+ */
+export const getSchemaFor = (dsnpType: DSNPType): Schema => {
+  const schemas = [GraphChangeSchema, BroadcastSchema, ReplySchema, ReactionSchema, ProfileSchema];
+
+  return schemas[dsnpType - 1];
```

Comment:
pretty slick!  the only thing I worry about is if the order of these in dsnp types ever changes.  hopefully the tests cover this, but I'd feel a little better if it were a key/value lookup.

Reply:
An interesting alternative would actually be to gain type safety and would throw an error when a new DSNPType was added (make sure not to use a `case default` and TypeScript will test for completeness on enums):

```suggestion  
export const getSchemaFor = (dsnpType: DSNPType): Schema => {
  switch(dsnpType) {
    case DSNPType.GraphChange:
      return GraphChangeSchema;
   // ...
  }
}
```